packages/react-atlas-core/src/avatar/Avatar.js

```diff
@@ -1,37 +1,57 @@
 import React, { PropTypes } from "react";
 import themeable from 'react-themeable';
 
+ var defaultImage = "http://www.jqueryscript.net/images/Simplest-Responsive-jQuery-Image-Lightbox-Plugin-simple-lightbox.jpg";
```

Comment:
i don't think default image belongs inside react-atlas-core. this is something that should be user defined in the app and sent down as a prop. we don't want a CDN image to be a dependency in our base library framework